dictionary/README.md

```diff
@@ -3,3 +3,6 @@ Please write a couple of sentences explaining your choice of restart
 strategy.
 
 
+I chose a restart strategy of :permanent, for the fact that each new
```

Comment:
# +2

I don't normally give bonuses for the correct answer, but you're one of only three people who did, and it just makes me so happy...